client/src/components/EmailAlertSignup.tsx

```diff
@@ -49,29 +49,27 @@ const BuildingSubscribeWithoutI18n = (props: BuildingSubscribeProps) => {
   const showEmailVerification = (i18n: any) => {
     return (
       <div className="building-subscribe-status">
-        <div>
-          <div className="status-title">
-            <AlertIconOutline />
-            <Trans>Email verification required</Trans>
-          </div>
-          <div className="status-description">
-            {i18n._(t`Click the link we sent to ${email}. It may take a few minutes to arrive.`)}
-          </div>
-          <button
-            className="button is-secondary is-full-width"
-            onClick={() => AuthClient.resendVerifyEmail()}
-          >
-            <Trans>Resend email</Trans>
-          </button>
-        </div>
+        <Alert type="info">
+          <Trans>Verify your email to start receiving updates.</Trans>
+        </Alert>
+        <Trans render="div" className="status-description">
+          Click the link we sent to {email}. It may take a few minutes to arrive.
```

Comment:
`{email}` is showing up as plaintext here!
<img width="688" alt="Screenshot 2024-03-05 at 2 27 49 PM" src="https://github.com/JustFixNYC/who-owns-what/assets/17574626/1b666e16-fd1c-46fb-9879-b503c3dfaf95">

Reply:
It's working on local so I'm pretty sure this is just that issue with not having done `lingui extract` yet, but I've been holding off until everything is merged into `email-alerts..`